### PR TITLE
[13.0] Refactor tests and misc fix/imp

### PR DESCRIPTION
**Pull request by simahawk**
@sebastienbeau this is similar to #46 but I need it to be done for v13 quickly.
It uses latest version of the tool. I improved tests a little bit too and `synchronize` method now gets called for real.
Can you review this?

Additional changes:
+ Bonus: I moved se.index.config to the base module as planned since ages :)
+ Cherry-picked commit from https://github.com/OCA/search-engine/pull/48

### CODE REVIEW

**Comment by simahawk** [comment]
still have adjust few things

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# TMP dependency to include https://github.com/akretion/odoo-test-helper/pull/1
+git+git://github.com/simahawk/odoo-test-helper.git@master#egg=odoo-test-helper
```
Comment (guewen) [inline test-requirements.txt]: @sebastienbeau can you help on this? :)

**Review by guewen — APPROVED** [review]

File: test-requirements.txt (hunk 1 above)
Comment (sebastienbeau) [reply test-requirements.txt]: I can work on it tomorrow  PM

**Review by i-vyshnevska — APPROVED** [review]

File: test-requirements.txt (hunk 1 above)
Comment (simahawk) [reply test-requirements.txt]: @sebastienbeau any update? This is blocking all other PRs.

Comment (simahawk) [reply test-requirements.txt]: should be fixed w/ last commit

**Pull request merged**